.template-lintrc.js

```diff
@@ -1,5 +1,5 @@
 'use strict';
 
 module.exports = {
-  extends: 'recommended'
+  extends: 'recommended',
```

Comment:
should be octane :P

Reply:
Ah, good call! Will do.

Reply:
This has zero impact on this addon b/c there are no templates to lint, so I just included it!